.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: ['master']
+
+jobs:
+  build:
+    name: Build and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java:
+          - adopt@1.8
+          - graalvm-ce-java11@20.3.0
```

Comment:
I believe it would be good to ensure the project works on multiple jdk versions. This one is using the JDK 11 but with **Graal** so it is like two birds with one stone.
What do you think?

Reply:
yeah, totally agree!